src/components/send-offer/OfferFormContext.tsx

```
@@ -0,0 +1,218 @@
+    
+    case "SET_ERROR":
+      return {
+        ...state,
+        errors: {
+          ...state.errors,
+          [action.field]: action.error
+        }
+      };
+    
+    case "CLEAR_ERROR":
+      const { [action.field]: _, ...restErrors } = state.errors;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)